src/malli/experimental/time.cljc

```diff
@@ -0,0 +1,82 @@
+(ns malli.experimental.time
+  (:refer-clojure :exclude [<=])
+  (:require [malli.core :as m]
+            #?@(:cljs
+                [["@js-joda/core" :as js-joda]
+                 ["@js-joda/timezone"]
+                 [goog.object :as gobj]]))
+  #?(:clj (:import (java.time Duration LocalDate LocalDateTime LocalTime Instant ZonedDateTime OffsetDateTime ZoneId OffsetTime ZoneOffset))))
+
+(defn <= [^Comparable x ^Comparable y] (not (pos? (.compareTo x y))))
+
+(defn -min-max-pred [_]
+  (fn [{:keys [min max]}]
+    (cond
+      (not (or min max)) nil
+      (and min max) (fn [x] (and (<= x max) (<= min x)))
+      min (fn [x] (<= min x))
+      max (fn [x] (<= x max)))))
+
+(defn -temporal-schema [{:keys [type class type-properties]}]
+  (m/-simple-schema
+   (cond->
+     {:type type
+      :pred (fn pred [x]
+              #?(:clj  (.isInstance ^Class class x)
+                 :cljs (instance? class x)))
+      :property-pred (-min-max-pred nil)}
+     type-properties
+     (assoc :type-properties type-properties))))
+
+#?(:cljs
+   (do
+     (def Period (gobj/get js-joda "Period"))
```

Comment:
`(.-Period js-joda)`

These are static properties on the JS object, so interop property access should be used.

Reply:
ah right, thanks!